src/platform/plugins/shared/usage_collection/README.mdx

```diff
@@ -497,9 +497,9 @@ There are a few ways you can test that your usage collector is working properly.
 
 ## FAQ
 
-1. **How should I design my data model?**  
+1. **How should I design my data model?**
    Keep it simple, and keep it to a model that Kibana will be able to understand. Bear in mind the number of keys you are reporting as it may result in fields mapping explosion. Flat arrays, such as arrays of strings are fine.
-2. **If I accumulate an event counter in server memory, which my fetch method returns, won't it reset when the Kibana server restarts?**  
```

Comment:
The extra spaces at the end are intentional because they affect formatting. Can we keep them, please?